README.md

```diff
@@ -2,30 +2,29 @@
 
 
 ## Overview
+OpenFin Notifications provides developers with a uniform way to create, display and organize desktop notifications as well as responding to notification events.
 
-OpenFin Notifications provide developers with a uniform way to create, display and organize desktop notifications as well as responding to notification events.
-
-Notifications will be displayed as toasts as well as being listed and organized in a Notification Center. The Notification Center can be accessed by clicking on the system tray notification icon.
+Notifications will be displayed as toasts as well as being listed and organized in a Notification Center. The Notification Center can be accessed by clicking on the Notifications icon in the system tray.
 
 This project consist of 3 parts:
-1. The Notifications Service, displaying and managing notifications and owning the Notification Center UI
-2. The Notifications Client, exposing API's for applications to create and manage notifications
+1. The Notifications Provider, displaying and managing notifications, and owning the Notification Center
```

Comment:
The term 'provider' gets used later on (previously without any introduction), so seemed worth introducing the term here.

Would be awkward to flip the other way - change uses of 'provider' to 'service' - given the ambiguity it would introduce.

Reply:
👍 from me on this.